testsuites/syncgateway/system/test_system_test.py

```diff
@@ -16,15 +16,15 @@
 
 # Set the default value to 404 - view not created yet
 SG_VIEWS = {
-    'sync_housekeeping': [
+    'sync_housekeeping_2.0': [
```

Comment:
can you add comment in system test to have TODO, to have conditional logic to verify sg_views based on sg version

Reply:
Done